Scripts/Source/OrgasmSystem.psc

```diff
@@ -0,0 +1,103 @@
+Scriptname OrgasmSystem hidden
```

Comment:
Not something like UD_OrgasmSystem?

Reply:
The idea was to make the system as its own mod in the future, but it doesn't look like it will happen right now. Nevertheless, it can be imagined as small mod inside UD